doc/nrf/protocols/matter/overview/bridge.rst

```diff
@@ -0,0 +1,67 @@
+.. _ug_matter_overview_bridge:
+
+Matter Bridge
+#############
+
+.. contents::
+   :local:
+   :depth: 2
+
+A Matter bridge is a device that makes it possible to use non-Matter IoT devices in a Matter fabric together with native Matter devices.
+A non-Matter devices used this way is called a *bridged device*.
+A Matter bridge exposes the bridged devices to Matter nodes in a fabric as Matter endpoints.
+This exposure is transparent, meaning that the Matter nodes are able to communicate with non-Matter bridged devices as if they were native Matter devices.
+
+It is out of scope for the Matter specification to define connectivity technologies used by the bridged devices, so they can use any technology (for example, Bluetooth® LE or Zigbee) as long as the Matter bridge supports it too.
+The Matter bridge translates between the specific subset of the Matter data model and the data representation specific for the other protocol.
+It is not the role of a bridge to translate all traffic between the protocols used on the networking layer.
```

Comment:
Maybe, we can state that it is up to product manufacturer (bridge manufacturer) to define which device types on Matter and which bridged devices it supports.